### PR TITLE
feat: add log decoder helper and import web3 [serverless]

### PR DESCRIPTION
**Pull request by EmperorOrokuSaki**

### CODE REVIEW

File: serverless/src/libs/log-decoder.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+var Web3 = require('web3');
+
+var web3 = new Web3(Web3.givenProvider || "ws://localhost:17895");
```
Comment (zoruka) [inline serverless/src/libs/log-decoder.ts:3]:
```suggestion
const web3 = new Web3(Web3.givenProvider || "ws://localhost:17895");
```

File: serverless/src/libs/log-decoder.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+export const logDecoder = (eventFieldsABI: { indexed: boolean; internalType: string; name: string; type: string; }[], data: string, topics: string[]) => {
+    return web3.eth.abi.decodeLog(eventFieldsABI, data, topics);
+};
```
Comment (zoruka) [inline serverless/src/libs/log-decoder.ts:7]:
```suggestion
export const logDecoder = web3.eth.abi.decodeLog;
```

**Review by jsonsivar — CHANGES_REQUESTED** [review]
As discussed in scrum, let's close this and add web3 export to our solidity helper and we can use the web3 decoder function directly from that exported web3 instance.

**Comment by EmperorOrokuSaki** [comment]
will be covered in #232

**Pull request closed without merging**